flight/Libraries/math/coordinate_conversions.c

```diff
@@ -366,6 +366,75 @@ void rot_mult(float R[3][3], const float vec[3], float vec_out[3], bool transpos
 }
 
 /**
+ * @brief LLA_linearization_float calculates linearized NED to LLA conversion using floats
+ * @param home_latitude is the home latitude times 10e6
+ * @param home_altitude
+ * @param T is the linearized NED to LLA conversion
+ */
+void LLA_linearization_float(int32_t home_latitude_1e7, float home_altitude, float T[])
```

Comment:
Might be called `LLA2NED_linearization...`. This way it is more explicit about what the operation is, and it makes space for my `ECEF2NED_linearization...`.